dnf-behave-tests/dnf/plugins-core/modulesync.feature

```diff
@@ -0,0 +1,117 @@
+@1868047
+Feature: Test for modulesync command
+
+
+Background:
+  Given I use repository "dnf-ci-fedora-modular"
+    And I use repository "dnf-ci-fedora"
+    And I enable plugin "modulesync"
+
+
+Scenario: I can download a module
+  Given I create directory "modulesync"
+    And I set working directory to "{context.dnf.installroot}/modulesync"
+    And I execute dnf with args "modulesync nodejs:8/minimal"
+   Then the exit code is 0
+    And file "modulesync/nodejs-8.11.4-1.module_2030+42747d40.src.rpm" exists
+    And file "modulesync/nodejs-8.11.4-1.module_2030+42747d40.x86_64.rpm" exists
+    And file "modulesync/nodejs-devel-8.11.4-1.module_2030+42747d40.x86_64.rpm" exists
+    And file "modulesync/nodejs-docs-8.11.4-1.module_2030+42747d40.noarch.rpm" exists
+    And file "modulesync/npm-8.11.4-1.module_2030+42747d40.x86_64.rpm" exists
+    And stdout is
+    """
+    <REPOSYNC>
+    Directory walk started
+    Directory walk done - 5 packages
+    Loaded information about 0 packages
+    Temporary output repo path: {context.dnf.installroot}/modulesync/.repodata/
+    Preparing sqlite DBs
+    Pool started (with 5 workers)
+    Pool finished
+    """
+
+
+Scenario: I can download a module using --downloaddir
+  Given I create directory "modulesync"
+    And I execute dnf with args "modulesync nodejs:8/minimal --downloaddir={context.dnf.installroot}/modulesync"
```

Comment:
May I ask you to add line `And I use repository "dnf-ci-fedora-modular-updates"` and update the set of downloaded packages? Then it will provide the difference in downloaded packages between scenario `Scenario: I can download a module with option newest only`